src/components/HistoryTables/ActorState/index.tsx

```diff
@@ -1,25 +1,17 @@
 import { FilecoinNumber } from '@glif/filecoin-number'
 import { useMemo, useState } from 'react'
-import styled from 'styled-components'
 import { useAddressQuery } from '../../../generated/graphql'
 import {
   useStateReadStateQuery,
   decodeActorCID,
   MsigState
 } from '../../../utils'
 import Box from '../../Box'
-import { P } from '../../Typography'
+import { Lines, Line } from '../../Layout'
 import { Title } from '../generic'
-import { DetailCaption, LineWrapper, Line } from '../detail'
+import { DetailCaption } from '../detail'
 import convertAddrToPrefix from '../../../utils/convertAddrToPrefix'
 
-const ViewState = styled(P).attrs(() => ({
-  color: 'core.primary',
-  role: 'button'
-}))`
-  cursor: pointer;
-`
-
```

Comment:
@Schwartz10 `p[role="button"]` styling has been added to `base-css` because it occurs in a few places. This makes these custom styled components redundant